### DISCUSSION TITLE
Deny direct access to theme PHP files?

### DISCUSSION THREAD

**Discussion by jmcintyresage** [bootscore/bootscore#366] [Ideas]
Shortly after public launch, one of our Bootscore-based sites started getting hit with requests for the base theme's index.php and search.php files, resulting in fatal PHP errors because get_header() wasn't defined.

Would the developers consider adding code that would deny direct access to the base theme's PHP files? Something as simple as:

```
// Stop if this file is called directly
defined( 'ABSPATH' ) || exit;
```

Respectfully,
JIm

**Comment by crftwrk** [comment]
Yes, this seems to be a good idea. Created an issue from this discussion https://github.com/bootscore/bootscore/issues/367 to track this. Do you want to create a PR?
